src/plays/savings-calculator/components/Total/total.scss

```diff
@@ -0,0 +1,59 @@
+@import "../../styles/variables";
+
+.total {
```

Comment:
@kd100100 , Can we scope these classes by adding play name prefix?

Reply:
Added play name to parent classes. Conflicts will be avoided!